ogr/services/github/service.py

```diff
@@ -178,3 +179,39 @@ def project_create(
     def get_pygithub_instance(self, namespace: str, repo: str) -> PyGithubInstance:
         token = self.authentication.get_token(namespace, repo)
         return PyGithubInstance(login_or_token=token)
+
+    def list_projects(
+        self,
+        namespace: str = None,
+        user: str = None,
+        search_pattern: str = None,
+        language: str = None,
+    ) -> List[GitProject]:
+
+        search_query = ""
+
+        if user:
+            search_query += f"user:{user}"
+
+        if language:
+            search_query += f" language:{language}"
```

Comment:
Just to check, will an empty space at the beginning of `search_query` work well in case of `user=None and langure="something"`?

Reply:
Yeah good question. I will take a look at that but I don't think it's going to be a problem.